client/app/services/intercom.js

```diff
@@ -23,7 +29,16 @@ export default Ember.Service.extend({
     /* eslint-enable camelcase */
   },
 
+  trackEvent(eventName) {
+    let eventAttribute = Ember.String.camelize(eventName);
+    if (!this.get(eventAttribute))  {
```

Comment:
Not a bad idea. Why don't we see how `trackEvents` is used and create the `trackEventOnce` event in the future if needed. At that point I can create the hash property for the service.